README.md

```diff
@@ -113,30 +114,32 @@ for _ in range(N_ITER):
     optimizer.step()
 ```
 
-# 🧭 Code Organization
+# Multi GPU User API
```

Comment:
we could say something like we support torch's FSDP for data parallelsim compose with Dtensors' TP API